packages/paste-website/src/components/callout/index.tsx

```diff
@@ -15,36 +16,45 @@ const borderColorPartial = (variant?: CalloutVariants): BorderColor => {
   return 'colorBorderPrimaryLight';
 };
 
+interface CalloutTitleProps {
+  as: Pick<HeadingProps, 'as'>;
+}
+
 const CalloutTitle: React.FC<CalloutTitleProps> = ({as = 'h3', children}) => (
   <Heading as={as} variant="heading50">
     {children}
   </Heading>
 );
 
-const CalloutText: React.FC<CalloutTextProps> = ({as = 'p', children, marginTop, marginBottom}) => (
-  <Text as={as} marginTop={marginTop} marginBottom={marginBottom}>
+const CalloutText: React.FC = ({children}) => (
+  <Text as="p" marginTop="space30">
```

Comment:
You may still need the override for `as` here for the homepage (and anywhere else) where we're passing `CalloutText` a list. https://github.com/twilio-labs/paste/pull/308/files#diff-3434f70a811aebf8c3d7a2a7bf6559beR98

You can't nest a list in a paragraph in HTML.

Reply:
Since `CalloutText` is intended as a Callout `p` wrapper, I removed it around the UnorderedList usage.

Reply:
Makes sense to me